collections/english-nouns.csv

```diff
@@ -2961,3 +2961,4 @@ merchandize,80839904129547128973182661428256906697287917300863854752579751281336
 kidnaper,69957007819168817257000086629248135067829535909481837772152367512202408838157
 microsphere,12430592501553366531366193527297595727176024872440078439288020661489291832438
 pentode,54801904824765800759815167047049974586957899357641518587936460357380582290087
+cartel,b05e621ee2e5e6d6119e0a987087ed32b6c6fe1113b181eaa85ede67faddbdd5

```

Comment:
Wait, how these was merged. It is not a token ID. @CryptoGatsu @Zimtente

Reply:
What do you mean ser?? If you run the code below it returns the hash provided. 

```
var namehash = require('@ensdomains/eth-ens-namehash');
var hash = namehash.hash('cartel')
hash = hash.replace('0x', '');
console.log('cartel' + "," + hash)
```

https://www.npmjs.com/package/@ensdomains/eth-ens-namehash

Please let me know if there is a mis-understanding on my end

Reply:
I was mistaken. The correct code for generating a valid hash is below 

```
const ethers = require('ethers')
const BigNumber = ethers.BigNumber
const utils = ethers.utils
const name = 'cartel'
const labelHash = utils.keccak256(utils.toUtf8Bytes(name))
const tokenId = BigNumber.from(labelHash).toString()
console.log('cartel' + "," + tokenId)
```

I will update this mr with correct hashes later today. Thanks for putting some eyes on this!

This should be the correct entry?

`
cartel,24930356794076149583364840017866382670785817347831594360562413046882721988203
`

Reply:
Please see my updated MR

https://github.com/Zimtente/ens-collections/pull/450

Reply:
New MR to fix hashes has been merged into main. Will resolve this conversation.